Lenddo/android-lenddo
# release v1.4.1

Lenddo SDK v1.4.1
 - Fix bug with metadata permission 
 - Update to Google sign in v2
 - Optimize imports and variables
 -  Update verifimelib
 - fix bug when getting onboarding service token
 - fix drop off points in data sdk
 - fix permission cancel bug
 - added permission events for cancelling permission screen
 - allow customization of permission screen
 - add a popup confirmation dialog upon cancelling the permission screen
 - fixed app title showing "Lenddo" instead of parent app's name
 - prevent data sdk from using a null application_id
 - minor bug fixes and code optimisations
 - Update documentation

Note: To update to the latest version:
1. Download the latest release version.
2. Remove the old module from your project Android Studio
3. Import the newly downloaded Lenddo Mobile SDK v1.4.1
4. Rebuild and run your application to confirm update
